### PR TITLE
[de] Update translations

### PR DESCRIPTION
**Pull request by typingbeaver**
Various additions and minor fixes

### CODE REVIEW

File: locales/de/_excludes.json [hunk 1]
```diff
@@ -99,7 +99,6 @@
     "Somalia",
     "Sri Lanka",
     "Status",
-    "St. Martin",
```
Comment (typingbeaver) [inline locales/de/_excludes.json]: Wrong exclude :facepalm:

File: docs/referents.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ de
 
 </td><td width="50%">
 
-@WhereIsLucas, @sotten
+@WhereIsLucas, @sotten, @typingbeaver
```
Comment (andrey-helldar) [inline docs/referents.md:55]: Sorry, this file contains information about the members of our team responsible for this translation.

So far, you have not been accepted into our team.

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

File: docs/referents.md (hunk 2 above)
Comment (typingbeaver) [reply docs/referents.md:55]: Ah, sorry, nevermind!

Comment (andrey-helldar) [reply docs/referents.md:55]:
```suggestion
@WhereIsLucas, @sotten
```

**Comment by andrey-helldar** [comment]
@sotten, can you check this PR?

**Review by sotten — APPROVED** [review]

**Comment by sotten** [comment]
> @sotten, can you check this PR?

sorry I was on vacation. I have not noticed any errors.

**Comment by andrey-helldar** [comment]
Danke

**Pull request merged**